src/client/Microsoft.Identity.Client/Platforms/netdesktop/NetDesktopCryptographyManager.cs

```diff
@@ -85,10 +92,21 @@ public byte[] SignWithCertificate(string message, X509Certificate2 certificate)
                 return rsa.SignData(messageBytes, sha);
             }
 #else
-            using (var key = certificate.GetRSAPrivateKey())
+            if (!CertificateToRsaMap.TryGetValue(certificate.Thumbprint, out RSA rsa))
```

Comment:
I assume the same code needs to run on .NET Core ? I remember you saying it's not going to work on NetStandard1.3

Reply:
I am not quite following....

Reply:
@bgavrilMS @brentschmaltz 
Should the same caching code be included in [NetCoreCryptographyManager](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/117256981411b9b0c017eb208f98c6259deaa52b/src/client/Microsoft.Identity.Client/Platforms/netcore/NetCoreCryptographyManager.cs#L76) and [NetStandard13CryptographyManager](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/117256981411b9b0c017eb208f98c6259deaa52b/src/client/Microsoft.Identity.Client/Platforms/netstandard13/NetStandard13CryptographyManager.cs#L68)?